app/controllers/campaign_controller.rb

```diff
@@ -1,33 +1,17 @@
 class CampaignController < ApplicationController
-  before_filter :redirect_if_api_request
+  include ApiRedirectable
+  include Previewable
+
   before_filter -> { set_expiry unless viewing_draft_content? }
```

Comment:
Definitely could be! I guess I was wondering if it was worth it for a one-liner like this...